### PR TITLE
Bug fixes for Deposit Item

### PR DESCRIPTION
**Pull request by murny**
First round of bug fixes:
- Increase upload file size to 1GB
- Reorder year dropdown, most recent first(2019) to least(1918)
- Fix weird edge case where on error submission where next step in progress bar is a clickable link
- Bandaid over a few weird tests, need revisiting
- Do not link related item in item view (broken anyways) (requirements will come later for this)

Will open another PR to tackle whats left:
TODO https://github.com/ualbertalib/jupiter/issues/269

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -244,7 +244,7 @@ GEM
     puma (3.11.2)
     pundit (1.1.0)
       activesupport (>= 3.0.0)
-    rack (2.0.3)
+    rack (2.0.4)
```
Comment (murny) [inline Gemfile.lock]: Need to bump this as this destroys your CPU on large file uploads. 

Original issue here: https://github.com/rack/rack/issues/1075

Now fixed in 2.0.4
